src/fairseq2/nn/utils/module.py

```diff
@@ -570,3 +571,44 @@ def get_module_size(module: Module) -> ModuleSizeInfo:
         info.total_size_bytes += size_bytes
 
     return info
+
+
+def normalize_truncate(
```

Comment:
I think we can use PyTorch's [trunc_normal_](https://pytorch.org/docs/stable/nn.init.html#torch.nn.init.trunc_normal_) instead of this function. This is also noted in the reference implementation here: https://github.com/facebookresearch/jepa/blob/main/src/utils/tensors.py#L18-L19